include/api/CSingleStreamSearcher.h

```diff
@@ -38,13 +38,19 @@ class API_EXPORT CSingleStreamSearcher : public core::CDataSearcher {
     //! called.
     CSingleStreamSearcher(const TIStreamP& stream);
 
+    //! Ensure destructor is called to consume stream to its end
+    ~CSingleStreamSearcher();
```

Comment:
```suggestion
    ~CSingleStreamSearcher() override;
```

(this doesn't affect the generated code, it's just a clue for readers like `@Override` in Java)